src/css/HomePage.css

```diff
@@ -0,0 +1,97 @@
+/* All h1 text elements. In this case, it's just the "The Jumpstart Blog" text. */
+h1 {
+  margin-left: 30px;
+}
+
+/* All button elements. In this case, it's the home and add post buttons. */
+button {
+  font-size: 115%;
+  height: 100px;
+  width: 200px;
+}
+
+/* A h4 element. We use it for the post author, comment count, and clap count. */
+h4 {
+  font-weight: normal;
+  padding-right: 5px;
```

Comment:
This seems like a weird thing to put on an element selector... Can we maybe use an "emoji" class?

Reply:
It's not just used for emojis (although there are emojis in there) so I named it `postFooterText`